waypoint_planner/src/velocity_set_lanelet2/velocity_set_lanelet2.cpp

```diff
@@ -644,7 +644,7 @@ int fillWaypointsNearestArea(VelocitySetPath& vs_path, const autoware_msgs::Lane
   double dx = pose.pose.position.x - next_waypoint.pose.pose.position.x;
   double dy = pose.pose.position.y - next_waypoint.pose.pose.position.y;
   double dz = pose.pose.position.z - next_waypoint.pose.pose.position.z;
-  double distance = sqrt(pow(dx, 2) + pow(dy, 2) + pow(dz, 2));
+  double distance = sqrt(pow(dx, 2) + pow(dy, 2));
```

Comment:
以下のようにした方がまとまりが良いと思いますがいかがでしょうか。
```suggestion
  double distance = hypot(dx, dy);
```

Reply:
同意です、対応しました